src/model/asset.js

```diff
@@ -21,7 +21,7 @@ class Asset {
                 }
 
                 if (query === '') {
-                    url += '&limit=100';
+                    url += '&limit=999';
```

Comment:
If I'm not wrong, there are other occurrences of `999` in the codebase. What do you think about introducing a constant with this limit and reusing it?

Reply:
Thanks for the suggestion Jonas, I was struggling with this

Reply:
Dona